docs/source/getting-started.md

```diff
@@ -0,0 +1,194 @@
+# Getting Started with ExecuTorch
+This section is intended to describe the necessary steps to take PyTorch model anda run it using ExecuTorch. To use the framework, you will need to take the following steps:
+- Install the ExecuTorch python package and runtime libraries.
+- Export the PyTorch model for the target hardware configuration.
+- Run the model using the ExecuTorch runtime APIs.
+
+<hr/>
+
+## Installation
+To use ExecuTorch, you will need to install both the Python package and the appropriate platform-specific runtime libraries.
+
+Pip is the recommended way to install the ExecuTorch python package. This package includes the dependencies needed to export a PyTorch model, as well as Python runtime bindings for model testing and evaluation.
+```
+pip install executorch
+```
+
+#### System Requirements
```

Comment:
Decouple the system requirements between host (development) vs target (deployment) platform